egs/wsj/s5/steps/cleanup/create_segments_from_ctm.pl

```diff
@@ -325,7 +325,7 @@ sub ProcessWav {
   # Save the aligned CTM if needed
   if(defined($ACT)){
     for (my $i=0; $i<=$#aligned_ctm; $i++) {
```

Comment:
I can't figure out what change is being made here, and this is a deficiency in the original code.  At a place like this it's nice to have an example, like
# print "my_recording1 0.1 0.5 some_word\n"
The CTM format has a well-defined number of fields.  It's not true that this code can be correct, and the old code could have been correct.    Perhaps you and the original author (@vimalmanohar ?) can clarify.